podman_compose.py

```diff
@@ -910,12 +911,9 @@ def get_net_args_from_network_mode(compose, cnt):
         aliases_on_container = [service_name]
         if cnt.get("_aliases"):
             aliases_on_container.extend(cnt.get("_aliases"))
-        net_args.append("--network=bridge")
-        mac_address = cnt.get("mac_address")
-        if mac_address:
-            net_args.append(f"--mac-address={mac_address}")
```

Comment:
I think we lose mac-address here. Is this deliberate? If yes, please split to separate commit with explanation.